python-main.js

```diff
@@ -1258,22 +1258,11 @@ function web_editor(config) {
         webusbErrorHandler(error);
     }
 
-    function clearDapWrapper(event) {
-        if(window.dapwrapper || window.previousDapWrapper) {
-            window.dapwrapper = null;
-            window.previousDapWrapper = null;
-        }
-    }
-
```

Comment:
Function removed from here, but code still present in webusbErrorHandler():

https://github.com/bbcmicrobit/PythonEditor/blob/0efa8a8dadc09161b89be3ce607621a7827b01a0/python-main.js#L1332-LL1342